packages/firestore/test/integration/api/database.test.ts

```diff
@@ -2245,4 +2246,177 @@ apiDescribe('Database', persistence => {
       });
     });
   });
+
+  describe('sort documents by DocumentId', () => {
+    it('snapshot listener sorts query by DocumentId same way as get query', async () => {
+      const testDocs = {
+        A: { a: 1 },
+        a: { a: 1 },
+        Aa: { a: 1 },
+        '7': { a: 1 },
+        12: { a: 1 },
```

Comment:
12 -> '12' ? Is this getting converted to a string in `withTestCollection`? Same question for usages of `12` below.

Reply:
yes, it does. i can wrap them with '' for precaution.